test/__snapshots__/test.js.snap

```diff
@@ -355,6 +355,357 @@ const Impl = require(\\"../implementations/LegacyArrayClass.js\\");
 "
 `;
 
+exports[`MixedIn.idl 1`] = `
```

Comment:
I guess this doesn't directly test that mixing something in to multiple places doesn't reuse the same object... but it does indirectly, as it'd be hard to imagine code generation for the second class that duplicates it.